activitysim/abm/models/tour_od_choice.py

```diff
@@ -0,0 +1,115 @@
+# ActivitySim
+# See full license in LICENSE.txt.
+import logging
+
+import pandas as pd
+
+from activitysim.core import tracing
+from activitysim.core import config
+from activitysim.core import inject
+from activitysim.core import pipeline
+from activitysim.core import simulate
+
+from activitysim.core.util import assign_in_place
+
+from .util import tour_od
+from .util import estimation
+
+logger = logging.getLogger(__name__)
+
+
+@inject.step()
+def tour_od_choice(
+        tours,
+        persons,
+        households,
+        land_use,
+        network_los,
+        chunk_size,
+        trace_hh_id):
+
+    """
+    Given a set of previously generated tours, each tour needs to have an
+    origin and a destination. In this case tours are the choosers, but
+    the associated person that's making the tour does not necessarily have
+    a home location assigned already. So we choose a tour origin at the same
+    time as we choose a tour destination, and assign the tour origin as that
+    person's home location.
+    """
+
+    trace_label = 'tour_od_choice'
+    model_settings_file_name = 'tour_od_choice.yaml'
+    model_settings = config.read_model_settings(model_settings_file_name)
+    origin_col_name = model_settings['ORIG_COL_NAME']
+    dest_col_name = model_settings['DEST_COL_NAME']
+    alt_id_col = tour_od.get_od_id_col(origin_col_name, dest_col_name)
+
+    sample_table_name = model_settings.get('OD_CHOICE_SAMPLE_TABLE_NAME')
```

Comment:
These settings need to be documented.

Reply:
Do you mean I should describe them in the docstring?